dev/ci/ci-common.sh

```diff
@@ -25,12 +25,16 @@ git_checkout()
   local _URL=${2}
   local _DEST=${3}
 
+  # Allow an optional 4th argument for the commit
+  local _COMMIT=${4:-FETCH_HEAD}
+  local _DEPTH=${5:-1}
```

Comment:
We could also say: if a specific commit is passed as an argument to `git_checkout` we clone the full repo. Since the use case is `coq-stdpp` and the repo is cloned in a few seconds anyways, this doesn't change much.

There might be a problem if the function is first called without the last arguments, then call again with them. The commit might be unreachable. But as this is fixed by manually deleting the clone and starting over.